com.unity.visualeffectgraph/Documentation~/Context-OutputSharedSettings.md

```diff
@@ -12,7 +12,7 @@ All outputs share these settings and property ports. The only exception is the S
 | --------------------------- | -------- | ------------------------------------------------------------ |
 | **Blend Mode**              | Enum     | Specifies the blending method for the system and determines how to use the alpha. The options are :<br/>&#8226; **Opaque**: Does not use the alpha value and renders the output in the opaque pass. This also writes to the depth buffer.<br/>&#8226; **Alpha**: Uses the [standard alpha blend](https://docs.unity3d.com/Manual/SL-Blend.html) mode and renders the output in the transparent pass. By default, this enables GPU sorting for particle outputs (which you can override using the **Sort** setting).<br/>&#8226; **Premultiplied Alpha**: Uses the [premultiplied alpha blend mode](https://docs.unity3d.com/Manual/SL-Blend.html) and renders the output in the transparent pass. By default, this enables GPU sorting for particle outputs (which you can override using the **Sort** setting).<br/>&#8226; **Additive**: Uses the [additive blend mode](https://docs.unity3d.com/Manual/SL-Blend.html) for alpha and renders the output in the transparent pass. |
 | **Use Alpha Clipping**      | Bool     | Indicates whether the system uses alpha clipping and discards pixels under a certain alpha threshold. If you enable this property, use the **Alpha Threshold** input to set the alpha value below which the system discards pixels. |
-| **Material Offset**         | Int      | **(Inspector)** An offset Unity applies to the rendering order. This setting corresponds to the Material [Sorting Priority](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@latest?subfolder=/manual/Renderer-And-Material-Priority.html) in HDRP and Material [Priority](https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@latest?subfolder=/manual/simple-lit-shader.html) in URP. It is equivalent to modifying the [Material.renderQueue](https://docs.unity3d.com/ScriptReference/Material-renderQueue.html) property. This value can be negative or positive. URP clamps this value between -50 and 50, and HDRP clamps this value between -100 and 100. |
+| **Sorting Priority**        | Int      | **(Inspector)** An offset Unity applies to the rendering order. This setting corresponds to the Material [Sorting Priority](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@latest?subfolder=/manual/Renderer-And-Material-Priority.html) in HDRP and Material [Sorting Priority](https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@latest?subfolder=/manual/simple-lit-shader.html) in URP. It is equivalent to modifying the [Material.renderQueue](https://docs.unity3d.com/ScriptReference/Material-renderQueue.html) property. This value can be negative or positive. HDRP and URP clamps this value between -50 and 50. |
```

Comment:
Thanks, when I remapped the offsets I didn't know they were referenced here too.

Reply:
Actually, I think we did both changes in parallel 😅